### PR TITLE
feat: added single video page

### PR DESCRIPTION
**Pull request by rohanmathur91**
## Description:
- Added single video page
- Onclicking on any video in the video listing page user will be navigated to a single video page 
- From a single video page, users can add/remove the video from watch later or from any playlist.

## Link:
https://evolt-prime.netlify.app/explore

## Screenshot (gif):
chrome-capture (18) (1)
mobile

### CODE REVIEW

File: src/components/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      try {
+        setLoader(true);
+        const {
+          data: { video },
+        } = await axios.get(`/api/video/${videoId}`);
+
+        setVideo(video);
+        setLoader(false);
+      } catch (error) {
+        console.log(error);
```
Comment (Aditi0505) [inline src/components/SingleVideo/SingleVideo.jsx:33]: You can display the error on the UI instead.

**Review by Aditi0505 — APPROVED** [review]
1. Good use of reusable components.
2. Clean and easily readable code.
3. Good naming convention used.

File: src/components/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (rohanmathur91) [reply src/components/SingleVideo/SingleVideo.jsx:33]: Yes, I will do that

**Review by Naveenchandar — APPROVED** [review]
File structure looks good
destructing all the available places makes code looks clean and precise.

**Pull request merged**